app/models/conversation.rb

```diff
@@ -151,6 +151,15 @@ def recent_messages
     messages.chat.last(5)
   end
 
+  def auto_resolve?
```

Comment:
this method won't be required, since are querying only the required objects in the job itself.